app/lib/katello/util/path_with_substitutions.rb

```diff
@@ -14,10 +14,14 @@ def initialize(path, substitutions)
         @resolved = []
       end
 
+      def split_path
+        @split ||= @path.split('/')
```

Comment:
nitpick: there's an attr_accessor for `path` (vs `@path`)

Reply:
I'm not sure what to do here (and maybe I'm missing some obvious ruby concept), there is an attr_accessor and you can set the path on initializing, which creates the instance variable `@path`. I'm not sure which is best to use, or should we even have both options?

Reply:
I like attr_accessor for instance variables primarily because it makes the code easy to change. Imagine if we didn't initialize `@path` anymore when creating the object. References to `@path` would break. If we were using `path` through an accessor we could then define a method called `path` which would do the new thing and the code would continue to work ie would could change the implementation of path seamlessly

Reply:
updated!

Reply:
Looks good 👍